### DISCUSSION TITLE
Incompatibility between versions in dependencies when clone and open the project and tried to install node models!

### DISCUSSION THREAD

**Discussion by ana54321** [driftyco/ionic-native#4568] [Q&A]
I need help solving the following problem when I try to do npm install:

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @awesome-cordova-plugins/email-composer@5.37.3
npm ERR! Found: @awesome-cordova-plugins/core@6.3.0
npm ERR! node_modules/@awesome-cordova-plugins/core
npm ERR! @awesome-cordova-plugins/core@"^6.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @awesome-cordova-plugins/core@"^5.1.0" from @awesome-cordova-plugins/email-composer@5.37.3
npm ERR! node_modules/@awesome-cordova-plugins/email-composer
npm ERR! @awesome-cordova-plugins/email-composer@"^5.37.3" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @awesome-cordova-plugins/core@5.46.0
npm ERR! node_modules/@awesome-cordova-plugins/core
npm ERR! peer @awesome-cordova-plugins/core@"^5.1.0" from @awesome-cordova-plugins/email-composer@5.37.3
npm ERR! node_modules/@awesome-cordova-plugins/email-composer
npm ERR! @awesome-cordova-plugins/email-composer@"^5.37.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Users\AppData\Local\npm-cache_logs\2023-05-15T15_10_49_437Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\AppData\Local\npm-cache_logs\2023-05-15T15_10_49_437Z-debug-0.log

In package.json I already have dependencies and their respective versions but when I clone the project I necessarily have to install the node_models and that's when the above error occurs. Someone can help me?

The dependencies that come in the project are these:

"dependencies": {
"@angular/common": "~12.0.1",
"@angular/core": "~12.0.1",
"@angular/forms": "~12.0.1",
"@angular/platform-browser": "~12.0.1",
"@angular/platform-browser-dynamic": "~12.0.1",
"@angular/router": "~12.0.1",
"@angular/service-worker": "~12.0.1",
"@awesome-cordova-plugins/core": "^6.3.0",
"@awesome-cordova-plugins/email-composer": "^5.37.3",
"@awesome-cordova-plugins/onesignal": "^5.43.0",
"@babel/polyfill": "^7.12.1",
"@capacitor/android": "^3.1.1",
"@capacitor/app": "1.0.2",
"@capacitor/browser": "1.0.7",
"@capacitor/core": "3.1.1",
"@capacitor/device": "^1.0.2",
"@capacitor/haptics": "1.0.2",
"@capacitor/ios": "3.1.1",
"@capacitor/network": "^1.0.2",
"@capacitor/share": "^1.0.3",
"@capacitor/splash-screen": "^1.0.2",
"@capacitor/status-bar": "^1.0.2",
"@capacitor/storage": "^1.0.3",
"@ionic-native/core": "^5.0.0",
"@ionic-native/facebook": "^5.33.1",
"@ionic-native/google-maps": "^5.5.0",
"@ionic-native/google-plus": "^5.33.1",
"@ionic-native/http": "^5.31.1",
"@ionic-native/in-app-browser": "^5.36.0",
"@ionic-native/native-geocoder": "^5.31.1",
"@ionic-native/network-interface": "^5.31.1",
"@ionic-native/onesignal": "^5.30.0",
"@ionic-native/photo-viewer": "^5.32.0",
"@ionic-native/spinner-dialog": "^5.36.0",
"@ionic/angular": "^5.5.2",
"capacitor-apple-login": "github:rlfrahm/capacitor-apple-login",
"com-sarriaroman-photoviewer": "^1.2.5",
"cordova-plugin-advanced-http": "^3.1.0",
"cordova-plugin-email-composer": "^0.10.0",
"cordova-plugin-facebook-connect": "^2.3.0",
"cordova-plugin-file": "^6.0.2",
"cordova-plugin-inappbrowser": "^5.0.0",
"cordova-plugin-native-spinner": "^1.1.3",
"cordova-res": "^0.15.3",
"firebase": "^8.2.5",
"jetifier": "^2.0.0",
"natives": "^1.1.6",
"onesignal-cordova-plugin": "^2.11.1",
"rxdb": "^8.0.0",
"rxjs": "^6.5.3",
"tslib": "^2.0.0",
"zone.js": "~0.11.4"
},

**Comment by fikretkurtiiev** [comment]
Have you tried what is written above?

_Fix the upstream dependency conflict, or retry this command with --force or --legacy-peer-deps_

Just try, usually helps:
**_npm i --legacy-peer-deps_**
